React/CoreModules/RCTRedBox.mm

```diff
@@ -29,7 +29,7 @@
 
 @class RCTRedBoxWindow;
 
-#if !TARGET_OS_OSX // TODO(macOS GH#774)
+#if !TARGET_OS_OSX // TODO:(macOS GH#774)
```

Comment:
Is there a new precedent for "TODO:" vs "TODO" ?

Reply:
It doesn't matter much but using the colon then Xcode recognizes it